Source/ThirdParty/GltfPipeline/addDefaults.js

```diff
@@ -508,8 +508,12 @@ define([
         options = defaultValue(options, {});
         addDefaultsFromTemplate(gltf, gltfTemplate);
         addDefaultTransformToAnimatedNodes(gltf);
-        addDefaultMaterial(gltf);
-        addDefaultTechnique(gltf);
+
+        if (gltf.asset.extras.gltf_pipeline_upgrade_10to20) {
+            addDefaultMaterial(gltf);
+            addDefaultTechnique(gltf);
+        }
```

Comment:
This is sort of a hack - this tells gltf-pipeline to only add default materials or techniques if the model was a 1.0 model. If the model has `pbrMetallicRoughness` these lines are not called. This was added to fix a test where 2 shaders were being created instead of 1.

This is temporary until https://github.com/AnalyticalGraphicsInc/cesium/pull/6805.